src/mixins/actionGlobal.js

```diff
@@ -23,6 +23,8 @@
 import Vue from 'vue'
 
 export default {
+	inheritAttrs: false,
```

Comment:
Why is this needed to add the unique class? (It is probably pretty obvious, but remember that I am new to Vue :-P )

Reply:
Argh, nice catch!
This is a wrong part of this pr :see_no_evil:

Reply:
Now I feel like a pro B-) :-P